.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
```

Comment:
This is missing from the required `.eslintrc.json` file ` "parser": "babel-eslint",`